words/words.md

```diff
@@ -358,117 +358,117 @@ But wait... haven't Taiwan and South Korea *already* contained COVID-19? For 4 w
+	* Les symptomatiques contribuent R = 0,8 (40%)
+	* Les pré-symptomatiques contribuent R = 0,9 (45%)
+	* Les asymptomatiques contribuent R = 0,1 (5%, quoique leur modèle a de l'incertitude et cela pourrait être bien plus bas)
+	* Les trucs environnementaux comme les poignées de porte contribuent R = 0,2 (10%)
 
-    And add up the pre- & a-symptomatic contacts (45% + 5%) and you get 50% of R!
+	Et ajoutez les contacts pré- et a-symptomatiques (45% et 5%) et vous obtenez 50% de R !
 
 <div class="sim">
 		<iframe src="sim?stage=int-4a&format=calc" width="285" height="340"></iframe>
 </div>
 
-Thus, even without 100% contact quarantining, we can get R < 1 *without a lockdown!* Much better for our mental & financial health. (As for the cost to folks who have to self-isolate/quarantine, *governments should support them* – pay for the tests, job protection, subsidized paid leave, etc. Still way cheaper than intermittent lockdown.)
+Ainsi, même sans mettre en quarantaine 100% des contacts, on peut obtenir R < 1 *sans confinement !* Bien mieux pour notre santé mentale et financière. (Et pour le coût pour les gens qui doivent s'auto-isoler ou se mettre en quarantaine, *les gouvernements devraient les soutenir* — payer pour les tests, la protection de l'emploi, subventionner les congés maladie, etc. C'est toujours bien moins cher que du confinement intermittent.)
 
-We then keep R < 1 until we have a vaccine, which turns susceptible <icon s></icon>s into immune <icon r></icon>s. Herd immunity, the *right* way:
+On garde alors R < 1 jusqu'à ce qu'on ait un vaccin, qui transforme les susceptibles <icon s></icon>s en immunisés <icon r></icon>s. L'immunité collective, de la *bonne* manière :
 
 <div class="sim">
 		<iframe src="sim?stage=int-4b&format=calc" width="285" height="230"></iframe>
 </div>
 
-(Note: this calculator pretends the vaccines are 100% effective. Just remember that in reality, you'd have to compensate by vaccinating *more* than "herd immunity", to *actually* get herd immunity)
+(Note : ce calculateur fait semblant que les vaccins sont efficaces à 100%. Rappelez-vous qu'en réalité, il faudrait compenser en vaccinant *plus* que « l'immunité collective », pour *vraiment* obtenir l'immunité collective)
 
-Okay, enough talk. Here's a simulation of:
+OK, assez parlé. Voici une simulation de :
 
-1. A few-month lockdown, until we can...
-2. Switch to "Test, Trace, Isolate" until we can...
-3. Vaccinate enough people, which means...
-4. We win.
+1. Un confinement de quelques mois, jusqu'à ce qu'on puisse…
+2. Passer à « Tester, Tracer, Isoler » jusqu'à ce qu'on puisse…
+3. Vacciner assez de gens, ce qui veut dire que…
+4. On a gagné.
 
 <div class="sim">
 		<iframe src="sim?stage=int-5&format=lines" width="800" height="540"></iframe>
 </div>
 
-So that's it! That's how we make an emergency landing on this plane.
+Et c'est tout ! Voilà comment on fait un atterrissage d'urgence avec cet avion.
```

Comment:
on fait --> réaliser ?

Reply:
make est un verbe très simple et il est conjugé avec "we" à mon avis à dessein (idée : on a tous participé à la réussite)

Reply:
ok